cli/kaos_cli/facades/workspace_facade.py

```diff
@@ -71,13 +75,15 @@ def delete(self):
         base_url = self.url
         name = self.workspace
 
+        print("in delete")
```

Comment:
Why `print` and not `click.echo`?

Reply:
Same, quick temporary change. Was left unremoved.